sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -345,17 +343,19 @@ class ScrollableBlock extends Component {
 }
 
 /// An optimized scrollable widget for a large number of children that are all
-/// of the same height. Use this widget when you have a large number of children
-/// or when you are concerned about offscreen widgets consuming resources.
+/// the same size in the scrollDirection. For example for ScrollDirection.vertical
+/// itemSize is the height of each item. Use this widget when you have a large
+/// number of children or when you are concerned about offscreen widgets
+/// consuming resources.
 abstract class FixedHeightScrollable extends Scrollable {
```

Comment:
This name is tricky, since we've decided to call the dimension you specify or "fix" extent. Calling the class FixedExtentScrollable doesn't sound right (particularly if the Scrollable's scrollOffset is renamed scrollExtent. Maybe RegularItemScrollable and VariableItemScrollable? I'm not feeling it :-).